native/packages/react-native-bpk-component-star-rating/index.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import BpkStarRating from './src/BpkStarRating';
+import BpkStar from './src/BpkStar';
+import { STAR_TYPES, getTypeByRating } from './src/star-types';
+
+export default BpkStarRating;
+export { BpkStar, STAR_TYPES, getTypeByRating };
```

Comment:
Not really a huge issue, but how come you're exporting `getTypeByRating`? I know it's being used in a test, but that should still work without exporting it in `index.js`.

Reply:
Fair point I'll remove it